flow-typed/tape-cup_v4.x.x.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable  */
```

Comment:
@rtsao - thoughts on either shipping this with [tape-cup](https://www.npmjs.com/package/tape-cup) or adding it to [flow-typed](https://github.com/flowtype/flow-typed)?  If we are the only consumers of _tape-cup_, I'd vote we ship it similar to what we do for `fusion-core`.